src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -552,6 +573,8 @@ public PortBinding apply(String s) {
     }
 
     public CreateContainerCmd fillContainerConfig(CreateContainerCmd containerConfig) {
+        HostConfig hostConfig = containerConfig.getHostConfig();
+
```

Comment:
Please remove these lines.
The established pattern in this code (and in the docker-java code) is that "everyone calls getHostConfig() every time they need it" rather than calling it once and reusing it.